docs/modules/plugins/partials/generic-ui-steps.adoc

```diff
@@ -117,3 +117,89 @@ When I wait until element located `$locator` disappears
 Scenario: Wait for element disappearance
 When I wait until element located `name(welcome-image)` disappears
 ----
+
+=== Get text of context element
+
+==== *_Info_*
+
+Extracts text of a context element and saves it to a variable
+
+==== *_Wording_*
+
+[source,gherkin]
+----
+When I set text found in search context to $scopes variable `$variableName`
+----
+
+==== *_Parameters_*
+
+. `$scopes` - xref:parameters:variable-scope.adoc[The set of variable's scope]
+. `$variableName` - name of a variable
+
+==== *_Usage_*
+
+.SaveText.story
+[source,gherkin]
+----
+Scenario: Get text of context element
+When I change context to element located `By.id(username)`
+When I set text found in search context to SCENARIO variable `username`
+----
+
+=== Get attribute value of context element
+
+==== *_Info_*
+
+Extracts attribute value of a context element and saves it to a variable
```

Comment:
describe what happens if context is not set to the element

Reply:
added warning about fail if context element is not set